### PR TITLE
[FEATURE, PARAM] updated default parameters of OpenPepXL

### PR DESCRIPTION
**Pull request by enetz**
OpenPepXL default parameters set to a very sensitive search.
Also added default modifications.

# Description

Please include a summary of the change and which issue is fixed.

# Checklist:
- [x] Make sure that you are listed in the AUTHORS file
- [x] Add relevant changes and new features to the CHANGELOG file
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
- [x] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

### CODE REVIEW

**Comment by timosachsenberg** [comment]
looks good

File: src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp [hunk 1]
```diff
@@ -74,28 +74,28 @@ using namespace OpenMS;
...
+    defaults_.setValue("precursor:min_charge", 2, "Minimum precursor charge to be considered.");
+    defaults_.setValue("precursor:max_charge", 8, "Maximum precursor charge to be considered.");
+    defaults_.setValue("precursor:corrections", ListUtils::create<int>("4, 3, 2, 1, 0"), "Monoisotopic peak correction. Matches candidates for possible monoisotopic precursor peaks for experimental mass m and given numbers n at masses (m - n * (C13-C12)). These should be ordered from more extreme to less extreme corrections. Numbers later in the list will be preferred in case of ambiguities.");
     defaults_.setSectionDescription("precursor", "Precursor filtering settings");
 
-    defaults_.setValue("fragment:mass_tolerance", 0.2, "Fragment mass tolerance");
-    defaults_.setValue("fragment:mass_tolerance_xlinks", 0.3, "Fragment mass tolerance for cross-link ions");
-    defaults_.setValue("fragment:mass_tolerance_unit", "Da", "Unit of fragment m");
+    defaults_.setValue("fragment:mass_tolerance", 20.0, "Fragment mass tolerance");
+    defaults_.setValue("fragment:mass_tolerance_xlinks", 20.0, "Fragment mass tolerance for cross-link ions");
```
Comment (cbielow) [inline src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp:83]: what is the unit of this?
does `fragment:mass_tolerance_unit` apply to both? (would be a bit unususal)

File: src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp [hunk 2]
```diff
@@ -74,28 +74,28 @@ using namespace OpenMS;
...
-    defaults_.setValue("precursor:min_charge", 3, "Minimum precursor charge to be considered.");
-    defaults_.setValue("precursor:max_charge", 7, "Maximum precursor charge to be considered.");
-    defaults_.setValue("precursor:corrections", ListUtils::create<int>("2, 1, 0"), "Monoisotopic peak correction. Matches candidates for possible monoisotopic precursor peaks for experimental mass m and given numbers n at masses (m - n * (C13-C12)). These should be ordered from more extreme to less extreme corrections. Numbers later in the list will be preferred in case of ambiguities.");
+    defaults_.setValue("precursor:min_charge", 2, "Minimum precursor charge to be considered.");
+    defaults_.setValue("precursor:max_charge", 8, "Maximum precursor charge to be considered.");
+    defaults_.setValue("precursor:corrections", ListUtils::create<int>("4, 3, 2, 1, 0"), "Monoisotopic peak correction. Matches candidates for possible monoisotopic precursor peaks for experimental mass m and given numbers n at masses (m - n * (C13-C12)). These should be ordered from more extreme to less extreme corrections. Numbers later in the list will be preferred in case of ambiguities.");
```
Comment (cbielow) [inline src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp:79]: just a note: you can do `{4, 3, 2, 1, 0}`

File: src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp (hunk 1 above)
Comment (enetz) [reply src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp:83]: @cbielow yes, the unit applies to both. Both tolerances are for fragments in the same spectrum, so should be very similar anyway. There are two tolerances mainly for older ion trap data, where people often use absolute numbers of 0.2 for most fragments and 0.3 is sometimes used for higher masses and cross-linked fragments are on the heavier side. When using relative tolerances, they should be the same.

File: src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp (hunk 2 above)
Comment (timosachsenberg) [reply src/openms/source/ANALYSIS/XLMS/OpenPepXLAlgorithm.cpp:79]: yep, getting rid of ListUtils::create<int> now works with modern C++

**Review by timosachsenberg — APPROVED** [review]

**Pull request merged**